swift/apple/FirezoneNetworkExtension/PacketTunnelProvider.swift

```diff
@@ -127,7 +111,7 @@ extension NEProviderStopReason: CustomStringConvertible {
     case .noNetworkAvailable: return "No network available"
     case .unrecoverableNetworkChange: return "Unrecoverable network change"
     case .providerDisabled: return "Provider disabled"
-    case .authenticationCanceled: return "Authentication cancelled"
+    case .authenticationCanceled: return "Authentication canceled"
```

Comment:
A few comments in the Rust code spelled this wrong too, it's one of the only words I always guess wrong for spelling.

Reply:
It is "cancelled" in British-English :)